lora/api/logging.go

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright (c) 2018
+// Mainflux
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package api
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/lora"
+	nats "github.com/nats-io/go-nats"
+)
+
+var _ lora.Service = (*loggingMiddleware)(nil)
+
+type loggingMiddleware struct {
+	logger logger.Logger
+	svc    lora.Service
+}
+
+// LoggingMiddleware adds logging facilities to the core service.
+func LoggingMiddleware(svc lora.Service, logger logger.Logger) lora.Service {
+	return &loggingMiddleware{
+		logger: logger,
+		svc:    svc,
+	}
+}
+
+func (lm loggingMiddleware) CreateThing(mfxThing string, loraDevEUI string) (err error) {
+	defer func(begin time.Time) {
+		message := fmt.Sprintf("create_thing - mfx_thing_%s/lora_dev_eui_%s took %s to complete", mfxThing, loraDevEUI, time.Since(begin))
```

Comment:
Please use same format as in other services: `Method A for this and that took...`.